build/prepare-sa-test.sh

```diff
@@ -9,7 +9,6 @@ KUBE_DEPLOY_SECRET_NAME=`kubectl -n $TESTING_NAMESPACE get sa infinispan-operato
 KUBE_API_EP=`kubectl get ep kubernetes -n default -o jsonpath='{.subsets[0].addresses[0].ip}'`
 KUBE_API_TOKEN=`kubectl -n $TESTING_NAMESPACE get secret $KUBE_DEPLOY_SECRET_NAME -o jsonpath='{.data.token}'|base64 --decode`
 
-cat deploy/role.yaml build/debug-role-patch.yaml | kubectl apply -n $TESTING_NAMESPACE -f -
```

Comment:
Ops, only `cat` need to be removed. Role should be kept

Reply:
Fixed